GNUmakefile

```diff
@@ -0,0 +1,90 @@
+# Since the current pony Makefile requires llvm-config to exist before
```

Comment:
after my experience with how the release process was broken previously by Makefile changes, I'm quite wary of this. We need to verify before any of this is merged that all release process related usage still works correctly.

Additionally, having GNUmakefile and Makefile in the same directory is confusing and once we have this working, we should figure out a better approach. This is a good "for now" approach that I'd like to address before merging.

Reply:
Agreed.